cdk/README.md

```diff
@@ -0,0 +1,23 @@
+# Infrastructure
+
+This directory defines the components to be deployed to AWS
+
+## Useful commands
+
+We follow the [`script/task`](https://github.com/github/scripts-to-rule-them-all) pattern,
+find useful scripts within the [`script`](./script) directory for common tasks.
+
+- `./script/setup` to install dependencies
+- `./script/start` to run the Jest unit tests in watch mode
+- `./script/lint` to lint the code using ESLint
+- `./script/test` to lint, run tests and generate templates of the CDK stacks
+- `./script/diff` to print the diff between a CDK stack and a running CloudFormation stack
+- `./script/ci` to run tests and synthesise the CDK template in CI
+
+There are also some other commands defined in `package.json`, including:
+
+- `yarn lint --fix` attempt to autofix any linter errors
+- `yarn format` format the code using Prettier
+- `yarn watch` watch for changes and compile
```

Comment:
(Annoying, but presumably we'll need to update this file here and elsewhere if we switch to npm and remove the script files.)

Reply:
Yep, I might just re-run project generation with the latest `cdk` and cherry-pick the other 3 commits!

Reply:
This has been simplified in https://github.com/guardian/mobile-save-for-later/pull/64/commits/76a530ac3054453d03cc18dca6a44f1f5f4c4896.